Directory.Packages.props

```diff
@@ -13,6 +13,11 @@
   </PropertyGroup>
   <ItemGroup>
     <!-- System packages -->
+    <PackageVersion Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="11.0.0" />
```

Comment:
These dependencies are temporary. I tried to build a local nuget version of the new RabbitMQ Stream Provider to a local solution but that gave me a lot of errors. Also tried to just import the project into a different solution but no lucky. So, for development productivity, I'm just creating a temporary project here to try stuff with the new provider.